build/src/lib.rs

```diff
@@ -108,7 +108,8 @@ pub fn generate_image_id_sol(guests: &[GuestListEntry]) -> Result<Vec<u8>> {
         .iter()
         .map(|guest| {
             let name = guest.name.to_uppercase().replace('-', "_");
-            let image_id = hex::encode(guest.image_id);
+            // NOTE: Works with both risc0-build v1 and v2
+            let image_id = hex::encode(bytemuck::cast::<_, [u8; 32]>(guest.image_id));
```

Comment:
This is an issue when using the latest `risc0-build` on `main` which bumps to v2 and changes the `image_id` field from `[u32; 8]` to `Digest`. This line is chosen to work with both versions.
https://github.com/risc0/risc0/commit/881e512732eca72849b2d0e263a1242aba3158af#diff-4bac0a581b7140eead855e0ed6debc8734ce2b49e3d5317f53fb13c5aa1335e3L169-R181